python/setup.py

```diff
@@ -292,7 +292,6 @@ def get_option_from_sys_argv(option_str):
           'build_py': build_py,
           'test_conformance': test_conformance,
       },
-      setup_requires = ['wheel'],
```

Comment:
I just added this line recently, so I think something must be wrong if it's getting removed again.

Reply:
Thanks for catching that, fixed.

There was one other small change that was somehow lost in the last integrate back to Piper. I'll keep a closer eye on the next integrate to Piper.